### PR TITLE
Release 1.18 performance data

### PR DESCRIPTION
**Pull request by AndreaM12345**
Performance data for 1.18

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [x] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
We should update the version here as well to 1.18.

File: archive/v1.17/docs/ops/performance-and-scalability/index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<!doctype html><html lang=en><head><title>Redirecting&mldr;</title><link rel=canonical href=/v1.17/docs/ops/deployment/performance-and-scalability/><meta name=robots content="noindex"><meta charset=utf-8><meta http-equiv=refresh content="0; url=/v1.17/docs/ops/deployment/performance-and-scalability/"></head><body><h1>Redirecting&mldr;</h1><a href=/v1.17/docs/ops/deployment/performance-and-scalability/>Click here if you are not redirected.</a></body></html>
```
Comment (ericvn) [inline archive/v1.17/docs/ops/performance-and-scalability/index.html:1]: I'm not sure why this file is included.

Comment (AndreaM12345) [reply archive/v1.17/docs/ops/performance-and-scalability/index.html:1]: wasn't sure if we wanted to do 1.17 data too

Comment (ericvn) [reply archive/v1.17/docs/ops/performance-and-scalability/index.html:1]: If we wanted 1.17 data, we should do it in a release-1.17 PR and then have it re-archived to 1.18 in another PR. At this point, I feel we can ignore that.

**Review by ericvn — APPROVED** [review]

**Pull request merged**